app/src/main/res/css/modules/column.sass

```diff
@@ -39,13 +39,15 @@
     margin-top: 5rem
     font-size: 1.3rem
   .items_container
-    overflow-x: hidden
-    overflow-y: scroll
-    overflow-x: hidden
+    position: relative
+    overflow: auto
```

Comment:
Nope, it's not. I think I know what you're referring to. Instead of `overflow-x`, this should be done:
https://stackoverflow.com/q/3058866/1454514

TL;DR: `word-wrap:break-word;`